### PR TITLE
adding property basics example

### PR DESCRIPTION
**Pull request by rahwang**
Part of #5300 

Along with the callback property example and interpolation example, this should cover the property system pretty well. I should be done with a property system tutorial soon too.

Can you take a look, @mramato ?

### CODE REVIEW

File: Apps/Sandcastle/gallery/Basic Properties.html [hunk 1]
```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Use the property system to define attribute values over time.">
+    <meta name="cesium-sandcastle-labels" content="">
```
Comment (mramato) [inline Apps/Sandcastle/gallery/Basic Properties.html]: Should this have the beginner label?

**Comment by mramato** [comment]
> I should be done with a property system tutorial soon too.

Is this example meant as a companion to the Tutorial?

**Comment by rahwang** [comment]
Hey @mramato I added a beginner tag. Yes, this is intended to go with the time-dynamic properties tutorial

**Comment by mramato** [comment]
OK, let's hold on this until the new tutorial is ready, since we may need to update the examples to match.

**Comment by hpinkos** [comment]
I'm going to close this since we won't have a chance to revisit this and the tutorial until later.

**Pull request closed without merging**